gce/notebook/Dockerfile

```diff
@@ -29,7 +29,14 @@ RUN conda install --yes -c conda-forge \
     zict \
     && conda clean -tipsy
 
-RUN jupyter labextension install @jupyter-widgets/jupyterlab-manager @jupyterlab/hub-extension
+# https://github.com/pangeo-data/pangeo/issues/145#issuecomment-371215319
+RUN conda install -c ioam/label/dev holoviews geoviews
```

Comment:
Just to be totally clear, up to date HoloViews and GeoViews releases do not yet exist. I'll ping you as soon as they are available.

Reply:
To keep the size of the image small you'll want to include the `&& conda clean -tipsy` bit from above.  You'll also want to verify that this doesn't bring down a lot of new packages and replace those above. 

If possible, I might suggest using `pip` (if holoviews is easy to install) or including this in the conda install line above.

Reply:
HoloViews should be easy to install but we haven't yet made any plans to make ``--pre`` releases on PyPI, certainly an option though. The real worry will be GeoViews which a) is not currently available using pip and b) has dependencies on the entire geostack (including GDAL, cartopy etc.), which is pretty heavy-weight.

Reply:
Well then we probably shouldn't install geoviews here if we can avoid it.  @rabernat are your needs satisfied with just holoviews?

Reply:
I am ok with just holoviews for now.

What if we just pip install holoviews from github master. Will this work?

Reply:
Actually after double checking it turns out I was wrong about GDAL, it only depends on ``geos`` and ``proj4`` which aren't too large.

Reply:
@philippjfr have you made releases yet? Or do you recommend I install from pip?

Reply:
Okay I can't say when that will be done as I'm not the one working on it, so maybe use pip after all.

Reply:
HoloViews 1.10.0a1 is now up on the ``ioam/label/dev`` channel.